src/main/java/net/nikr/eve/jeveasset/io/local/SettingsWriter.java

```diff
@@ -442,6 +446,39 @@ private void writeTablesViews(final Document xmldoc, final Map<String, Map<Strin
 		}
 	}
 
+	private void writeTablesFormula(final Document xmldoc, final Map<String, List<Formula>> formulas) {
+		Element tableFormulasNode = xmldoc.createElementNS(null, "tableformulas");
+		xmldoc.getDocumentElement().appendChild(tableFormulasNode);
+		for (Map.Entry<String, List<Formula>> entry : formulas.entrySet()) {
+			Element formulasNode = xmldoc.createElementNS(null, "formulas");
+			setAttribute(formulasNode, "tool", entry.getKey());
+			tableFormulasNode.appendChild(formulasNode);
+			for (Formula formula : entry.getValue()) {
+				Element formulaNode = xmldoc.createElementNS(null, "formula");
+				setAttribute(formulaNode, "name", formula.getColumnName());
+				setAttribute(formulaNode, "expression", formula.getOriginalExpression());
```

Comment:
Do we need to escape reserved characters here or with the setAttribute handle that? for example what happens if someone somehow puts a " in their expression?

Reply:
The library writing the XML does handle escaping everything.